orc8r/cloud/go/services/orchestrator/servicers/gaugeconverter.go

```diff
@@ -36,17 +35,16 @@ func convertFamilyToGauges(baseFamily *prometheus_models.MetricFamily) []*promet
 	gaugeFamilies := make([]*prometheus_models.MetricFamily, 0)
 	switch *baseFamily.Type {
 	case prometheus_models.MetricType_GAUGE:
-		gaugeFamilies = append(gaugeFamilies, baseFamily)
+		return append(gaugeFamilies, baseFamily)
 	case prometheus_models.MetricType_COUNTER:
-		gaugeFamilies = append(gaugeFamilies, counterToGauge(baseFamily))
+		return append(gaugeFamilies, counterToGauge(baseFamily))
 	case prometheus_models.MetricType_HISTOGRAM:
-		gaugeFamilies = append(gaugeFamilies, histogramToGauges(baseFamily)...)
+		return append(gaugeFamilies, histogramToGauges(baseFamily)...)
 	case prometheus_models.MetricType_SUMMARY:
-		gaugeFamilies = append(gaugeFamilies, summaryToGauges(baseFamily)...)
-	case prometheus_models.MetricType_UNTYPED:
-		gaugeFamilies = append(gaugeFamilies, untypedToGauge(baseFamily))
+		return append(gaugeFamilies, summaryToGauges(baseFamily)...)
+	default: // Untyped
+		return append(gaugeFamilies, untypedToGauge(baseFamily))
```

Comment:
Yeah this is just an unrelated cleanup that I ended up putting in. Also the append is required because some return a single family and some return arrays so I just do this to keep everything consistent. I could change it to creating arrays in place where required but I like the consistency here